Clinical-Genomics/scout
# Speed up variant page loading

This PR speeds up variant page loading (fix #1337)

**How to test**:
1. This branch is already on stage (you can also test before and after installing the branch)
1. Go to whichever variant and notice that page loads faster
1. Go to a variant that is causative also in another case: https://scout-stage.scilifelab.se/cust000/15026-miptest/e67092207773c559b7cbdbb6d4c521a9

**Expected outcome**:
1. Variant pages should load faster
1. Same variant causative in 2 cases should show correct link to causative variant in other case on top of variant page.

**Review:**
